sdk/python/packages/flet-core/src/flet_core/text.py

```diff
@@ -137,7 +138,7 @@ def __init__(
         size: OptionalNumber = None,
         weight: Optional[FontWeight] = None,
         italic: Optional[bool] = None,
-        style: Optional[TextThemeStyle] = None,
+        style: Optional[Union[TextThemeStyle, TextStyle]] = None,
```

Comment:
Nice idea! I'm not a big fan of mixing types though, but let's users decide! :)

Reply:
I think we might need to separate props... perhaps create `text_style` and `theme_style`?

Reply:
Since @ndonkoHenri noticed this 🫣 I'm not going to try convincing him that "it's OK like that" - let's have two separate properties 😏

Now, I'd love to have `style` property be of `TextStyle` to look the same as `TextSpan.style`, and then `theme_style` for named styles.

However, we don't want to break a lot of existing applications. I'm thinking what if we "temporary" make `style` property accepting both `TextThemeStyle` and `TextStyle` (like in this PR) and also add `theme_style` which accepts only `TextThemeStyle`. Will update docs for `.style` property to allow only `TextStyle`. What do you guys think?

Reply:
@50Bytes-dev would you implement that (along with previous note) please?